src/core/lombok/eclipse/handlers/HandleSuperBuilder.java

```diff
@@ -272,8 +272,9 @@ void setBuilderToAbstract() {
 		};
 		if (extendsClause instanceof QualifiedTypeReference) {
 			QualifiedTypeReference qualifiedTypeReference = (QualifiedTypeReference)extendsClause;
-			String superclassClassName = String.valueOf(qualifiedTypeReference.getLastToken());
-			String superclassBuilderClassName = job.replaceBuilderClassName(superclassClassName);
+			char[] superclassClassName = qualifiedTypeReference.getLastToken();
+			String builderClassNameTemplate = BuilderJob.fixBuilderClassName(annotationNode, "");
```

Comment:
Is there any reason you used `""` instead of `null` for the override parameter?

I also think we should rename the `fixBuilderClassName` method, it doesn't fix anything it just returns the current configured builder name. I don't think that this has to be done in this PR.

Reply:
I don't know any more, but I probably sort of copied it from line 116.

Reply:
Now using `null`instead of an empty string. And I renamed the method anyway. :)

Reply:
Thanks, I think you missed to rename the method in the eclipse handler :smile:

Reply:
Next try... ;)